types/remarkable/package.json

```diff
@@ -5,13 +5,12 @@
     "projects": [
         "https://github.com/jonschlinkert/remarkable"
     ],
-    "dependencies": {
+    "dependencies": {},
+    "devDependencies": {
         "@types/highlight.js": "^9.7.0",
+        "@types/remarkable": "workspace:.",
         "highlight.js": "^9.7.0"
```

Comment:
It seems wrong to have both the types and the real package. Does one of these need removal?

Reply:
thanks for reviewing, I view this PR as a strict improvement on the current state of things. I don’t think we should block it on the basis it might not solve other problems with this package.

Reply:
I don't know what you mean, I just suggested removing this line because it can't affect the tests.